### ISSUE TITLE
Paramétrage comptable par enseigne

### ISSUE THREAD

**Issue by paul-kilya** [CroissanceCommune/autonomie#704]
Sur les fiches des enseignes, en plus du code analytique, il faudrait pouvoir renseigner / surcharger le paramétrage de certains comptes utilisés ensuite dans les exports afin qu'ils se renseignent par défaut.

A première vue j'y mettrais : 

- Compte client général
- Compte client tiers
- Compte de banque

**Comment by eheliot12** [comment]
#667 pour certains taux de contributions

**Comment by sbodrero** [comment]
Cette fiche correspond à quelle vue ?

**Comment by eheliot12** [comment]
dans le parametrage des enseignes
par exemple https://demo.autonomie.coop/companies/701?action=edit

**Comment by sbodrero** [comment]
La longueur de code est connue (nb caractères) ?

**Comment by paul-kilya** [comment]
Non, ça dépend des logiciels de compta et de l'organisation de chaque CAE.

Pour les stocker en bdd on a un peu de tout apparemment (de 20 à 125 caractères). En tous cas il ne faut pas limiter la saisie.

**Comment by sbodrero** [comment]
Très bien merci @paul-kilya

**Comment by tonthon** [comment]
- [ ] Ajout des informations dans le modèle Company
- [ ] Ajout des informations dans les formulaires, vues
- [ ] Utilisation des informations dans les exports comptables
- [ ] Utilisation des informations dans l'enregistrement des encaissements
